src/shipit_static_analysis/shipit_static_analysis/report/base.py

```diff
@@ -16,6 +17,7 @@
 C/C++ static analysis found {defects_total} in this patch{extras_comments}.
  - {defects_tidy} found by clang-tidy
  - {defects_format} found by clang-format
+ - {defects_linter} found by mozlint
 
 You can run this analysis locally with: `./mach static-analysis check path/to/file.cpp` and `./mach clang-format -p path/to/file.cpp`
```

Comment:
Nit: The "run this locally" instructions only work to reproduce `clang-tidy` issues. The `mozling` line goes like this:

```bash
./mach lint path/to/file
```

Maybe we could tweak the tutorial like below?

```
You can run this analysis locally with:.
  - C/C++: `./mach static-analysis check path/to/file`
  - JS/Python: `./mach lint path/to/file`
```